roles/ceph-common/defaults/main.yml

```diff
@@ -250,6 +250,7 @@ monitor_interface: interface
 monitor_address: 0.0.0.0
 mon_use_fqdn: false # if set to true, the MON name used will be the fqdn in the ceph.conf
 monitor_address_block: false
+mon_pg_warn_max_per_osd: 300
```

Comment:
No no no, for this you have to use `ceph_conf_overrides` :), see https://github.com/ceph/ceph-ansible/blob/master/group_vars/all.yml.sample#L313-L328